proto/ligato/vpp/l3/l3xc.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package ligato.vpp.l3;
+
+option go_package = "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/l3;vpp_l3";
+
+message L3XC {
```

Comment:
I would suggest to use the name `L3XConnect` - so that:
- users of the API have some more clue on what it actually is (especially if it lacks any doc comments)
- it is more in line with [L2 xconnect](https://github.com/ligato/vpp-agent/blob/master/proto/ligato/vpp/l2/xconnect.proto) naming